sklearn/manifold/tests/test_t_sne.py

```diff
@@ -480,11 +496,12 @@ def test_init_not_available():
         tsne.fit_transform(np.array([[0.0], [1.0]]))
```

Comment:
Here also it would be interesting to check the dtype (and shape) of the output.

Reply:
It looks like this just tests that the call errs.

Reply:
I agree with @jjerphan. We can keep the test to be focused on the error checking.